lib/elixir/src/elixir_rewrite.erl

```diff
@@ -207,6 +207,10 @@ rewrite(?map, fetch, [Map, Key]) ->
   {maps, find, [Key, Map]};
 rewrite(?map, 'fetch!', [Map, Key]) ->
   {maps, get, [Key, Map]};
+rewrite(?map, 'get', [Map, Key]) ->
```

Comment:
Should the implementation of `Maps.get/3` use `:maps.get(key, map, default)` then? 

I was looking at past PRs, and [this was not inlined](https://github.com/elixir-lang/elixir/pull/6205#discussion_r120950701) in the past because `get/3` was not a BIF. I checked Erlang's source code, and it seems [it is not included in the `%% BIFs` section](https://github.com/erlang/otp/blob/master/lib/stdlib/src/maps.erl#L23).

I've also made some benchmarks, and apparently Elixir's implementation is faster than Erlang's.

Reply:
Good catch. I will revert this. The only reason I added it here is because of bootstrap issues but performance should be the main criteria.